offchain/advance-runner/src/broker.rs

```diff
@@ -98,47 +105,63 @@ impl BrokerFacade {
 
         tracing::trace!(rollups_claim.epoch_index,
             ?rollups_claim.epoch_hash,
-            "producing rollups claim"
+            "adding rollups claim to stream"
         );
 
-        let result = self
+        let last_claim_event = self
             .client
-            .peek_latest(&self.claims_stream)
+            .read_latest(&self.claims_stream)
```

Comment:
I think we should keep the old name here:
```suggestion
            .peek_latest(&self.claims_stream)
```